specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -340,7 +272,10 @@
             ]
           }
         }
-      }
+      },
+      "required": [
```

Comment:
These are models that are GET only right?  I don't see a PUT subassessment API.  Why change this from readonly to required (which is a breaking change if the user provides this model)

Reply:
currently this is read only,
removed the required for now

Reply:
removed the required, but the test fails on "discriminator must be a required property."
we would like that the object will be converted successfully and not remove the discriminator value.
re-added the required again for now